project/AkkaDisciplinePlugin.scala

```diff
@@ -24,6 +24,7 @@ object AkkaDisciplinePlugin extends AutoPlugin {
     if (enabled) {
       Seq(
         Test / scalacOptions --= testUndicipline,
+        Test / scalacOptions ++= testsExtraUndicipline,
```

Comment:
going the extra mile. Nice!

Reply:
I think it should have been `testsExtraDicipline`. We are adding one more rule, not removing it. Or am I missing something?

And shouldn't we have it also in compile scope?